reference_docs/YAML_models/ietf_v2_document_attributes.yaml

```diff
@@ -0,0 +1,524 @@
+    value_spec: boolean
+    value_default: true
+
+  inline-definition-lists:
+    tags: ["Global options"]
+    title: "Inline definition lists"
+    description: |
+      'Only applies to v2. By default, `<vspace blankLines="1">` is inserted after the definition in a v2 definition list, to satisfy the requirement from `idnits` validation that definition terms be separated by a carriage return from the definition. That is, by default inline definition lists are rendered as paragraphing definition lists. If the option is on, the additional `vspace` element is not added, and inline definition lists are left as is.'.
+    value_spec: boolean
+    value_default: false
+
+  flush-caches:
+    tags: ["Global options"]
+    title: "Flush caches"
+    description: |
+      "Delete and reload the caches of references to be included externally, and of workgroups, during processing of this document.\n
+      The caches are stored in `~/.metanorma-ietf-biblio-cache.json` and `~/.metanorma-ietf-workgroup-cache.json`.".
+    value_spec: boolean
+    value_default: false
+
+  biblio-dir:
+    tags: ["Global options"]
+    title: "Reference directory"
+    description: "Name of directory. If present, gives the name of a directory from which RFC XML references are to be read into the document, rather than assuming the references are already present in the document.".
+
+  normative:
+    tags: ["Global options"]
+    title: "Normative"
+    description: 'Comma-delimited list of reference anchors. Used in conjunction with `:biblio-dir:`, which uses a single directory for all references: this attribute lists those references which are to be considered normative, and listed under the "`Normative References`" heading.'.
+    value_spec: comma-list
+
+#--- Document attributes  ---#
+  title:
+    tags: ["Document attributes"]
+    title: "Title of document"
+    description: "Title of document.".
+    required: true
+
+  abbrev:
+    tags: ["Document attributes"]
+    title: "Abbreviation of document"
+    description: "Abbreviation of document title. Usually the document name without the keyword `draft-`. RFC XML: `rfc/front/title@abbrev`".
+    required: true
+
+  name:
+    tags: ["Document attributes"]
+    title: "Name of document"
+    description: |
+      "Sets the document's name. This should be a number if
+      the document is an RFC, and a name (in the form of `draft-ietf-somewg-someprotocol-07`)
+      if it is an Internet-Draft.
+      When `doctype` is set to:
+
+      * `internet-draft`: the value should be in the form `draft-ietf-somewg-someprotocol-07`.
+      ** RFC XML: `rfc@docName`
+      * `rfc`: the value should be a number like `7991` as described
+      in  https://tools.ietf.org/html/rfc7991#section-2.47.6[here]
+      RFC XML: `rfc@number`
+
+      NOTE: Usage here differs from usage in AsciiRFC v3.".
+    required: true
+
+  status:
```

Comment:
They do do exactly the same. We have them because I was wanting to be backward compatible with something I'd released three years ago. docstage should be preferred, since it corresponds to the XML markup: status is in general a compound element, consists of stage, optional substage, and optional iteration.